internal/constellation/state/state.go

```diff
@@ -168,6 +171,13 @@ type Azure struct {
 	AttestationURL string `yaml:"attestationURL"`
 }
 
+// OpenStack describes the infra state related to OpenStack.
+type OpenStack struct {
+	// description: |
+	//   ID of the network
+	NetworkID string `yaml:"networkID"`
```

Comment:
Is there any validation we should run on this field (CLI and Terraform provider), or is this mostly optional?

Reply:
I don't think validation would be useful. This is currently output generated by the embedded terraform for OpenStack. There is no Constellation terraform provider support for OpenStack.